Adding device specific define files

chayan-04 opened the pull request:
Since structure definition is different
for different AP model, creating different
definition file here for different board.

## Summary
<!-- Mandatory
Explain here the why, the rationale and motivation, for the changes.
-->

## Checklist
<!-- Mandatory
Please confirm the following by replacing the space with an "x" between the []:
-->

- [ ] No AI generated code was used in this PR

## Related issues
<!-- Mandatory
All PRs should resolve an issue, if one does not exist, please open one.
-->

resolves #

chayan-04 committed:
Adding device specific define files

Since structure definition is different
for different AP model, creating different
definition file here for different board.

chayan-04 committed:
Changing duplex and speed to string

Earlier integer is used in speed and
duplex for interface stats in ah_wireless.
Changing it as per nos-API to string.

arnelplim commented on plugins/inputs/ah_wireless/ah_wireless.go:
```diff
@@ -2746,30 +2746,30 @@ func Gather_EthernetInterfaceStats(t *Ah_wireless) error {
 		if (link_status == ETH_SET_MII_LINK_DOWN) {
-			t.ethx_stats[i].duplex = 0
-			t.ethx_stats[i].speed = 0
+			t.ethx_stats[i].duplex = "LINK_DOWN"
```
Is setting these to "LINK_DOWN" typical?  If these are statically set, shouldn't you display those settings even when link is down?

chayan-04 replied:
@arnelplim , No we are reading data using ioctl. Earlier, server expected integer. Now they have changed api spec to string. Thats why changing them.

chayan-04 committed:
ah_wireless: Proper device stats

Because of datatype mismatch string scan
was returing everything as 0. Fixed this
issue here.

chayan-04 committed:
telegraf ntp server error

ntp server information was read from
wrong structure. Corrected it here.
